int-ui/int-ui-standalone/src/main/kotlin/org/jetbrains/jewel/intui/standalone/PainterProvider.kt

```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.jewel.intui.standalone
+
+import org.jetbrains.jewel.painter.ResourcePainterProvider
+
+/**
+ * Create [PainterProvider][org.jetbrains.jewel.painter.PainterProvider] for show standalone module resource.
```

Comment:
```suggestion
 * Create a [PainterProvider][org.jetbrains.jewel.painter.PainterProvider] to load a
 * resource from the classpath.
```

I would probably call this function `resourcePainterProvider` since it's not important that it is in the standalone module.